docs/website/docs/general-usage/credentials/setup.md

```diff
@@ -16,6 +16,10 @@ It then [injects](advanced/#injection-mechanism) these values where needed in fu
 
 ## Available config providers
 
+:::tip dlt+
+If you want to define all sources, destinations, pipelines and parameters in the `yaml` file, check out [dlt+](../../plus/features/projects.md).
```

Comment:
```suggestion
To define your configuration (including sources, destinations, pipeline and parameters) in a declarative way in the YAML file, check out [dlt+](../../plus/features/projects.md).
```

Let's stress that this is not only YAML file but also a declarative style of definition.